src/components/coin-row/BalanceCoinRow.js

```diff
@@ -24,7 +24,7 @@ const formatPercentageString = percentString =>
   percentString ? percentString.split('-').join('- ') : '-';
 
 const BalanceCoinRowCoinCheckButton = styled(CoinCheckButton).attrs({
-  isAbsolute: true,
```

Comment:
There's a few references to the `isAbsolute` prop but it doesn't seem to be used anymore now that this line has been changed? Should it be cleaned up?

Reply:
yes